hudi-common/src/test/java/org/apache/hudi/common/fs/inline/TestInLineFileSystem.java

```diff
@@ -369,7 +369,8 @@ private Path getRandomInlinePath() {
   private void verifyFileStatus(FileStatus expected, Path inlinePath, long expectedLength, FileStatus actual) {
     assertEquals(inlinePath, actual.getPath());
     assertEquals(expectedLength, actual.getLen());
-    assertEquals(expected.getAccessTime(), actual.getAccessTime());
+    // removing below assertion as it is flaky on rare occasion (difference is in single-digit ms)
```

Comment:
Let's either create a ticket to follow-up or remove completely

Reply:
I think we can remove it as asserting access time does not serve any purpose for this test. All we care about is length and block size. Anyway, we have the modification time assetion in the end.